### PR TITLE
Removed hard-coded constants from Mutators generator test 

### PR DESCRIPTION
**Pull request by sanmai**
This PR:

- [x] Covered by tests

### CODE REVIEW

**Comment by BackEndTea** [comment]
:+1: This is cleaner than my solution.

File: tests/Mutator/Util/MutatorsGeneratorTest.php [hunk 1]
```diff
@@ -13,17 +13,29 @@
 use Infection\Mutator\Arithmetic\Plus;
 use Infection\Mutator\Boolean\FalseValue;
 use Infection\Mutator\Boolean\TrueValue;
+use Infection\Mutator\Util\MutatorProfile;
 use Infection\Mutator\Util\MutatorsGenerator;
 use PHPUnit\Framework\TestCase;
 
 class MutatorsGeneratorTest extends TestCase
 {
+    private function countMutatorsFromDefaultProfile()
```
Comment (BackEndTea) [inline tests/Mutator/Util/MutatorsGeneratorTest.php]: I'd move this to a `setUp` function.

Comment (sanmai) [reply tests/Mutator/Util/MutatorsGeneratorTest.php]: Done.

File: tests/Mutator/Util/MutatorsGeneratorTest.php [hunk 2]
```diff
@@ -13,17 +13,29 @@
...
 use Infection\Mutator\Util\MutatorsGenerator;
 use PHPUnit\Framework\TestCase;
 
 class MutatorsGeneratorTest extends TestCase
 {
+    private $countDefaultMutators;
+
+    protected function setUp()
+    {
+        $this->countDefaultMutators = 0;
```
Comment (theofidry) [inline tests/Mutator/Util/MutatorsGeneratorTest.php]: from what I see the list doesn't change between each test so either you should initialize the value only when `$countDefaultMutators === null` or do that in the `beforeClass` something method which is called only once

Comment (sanmai) [reply tests/Mutator/Util/MutatorsGeneratorTest.php]: Indeed it doesn't. Done.

**Comment by borNfreee** [comment]
Thank you @sanmai

**Pull request merged**